src/exprestive.coffee

```diff
@@ -47,6 +47,23 @@ class Exprestive
       @controllers[camelCase controllerName][controllerAction] arguments...
 
 
+  # Returns if the passed filePath resembles a controller
+  # Controllers are .js or .coffee files that export a constructor and name
+  fileIsController: (filePath) ->
```

Comment:
This could be extracted into a dedicated class, so that you can test this. There are tons of edge cases to be sure about here.

```coffeescript
# Recognizes whether a given file is an Exprestive controller
class ControllerDetector

  constructor: (filePath) ->

  # Returns whether the file is an Expressive controller
  isController: ->
    isJsFile() and exportsName() and exportsConstructorFunction() and nameEndsWithController()

  isJsFile: ->
    extension = path.extname filePath
    extension is '.coffee' or extension is '.js'

  exportsName: ->
    maybeController = require filePath
    maybeController.name?

  exportsConstructorFunction: ->
    maybeController = require filePath
    typeof maybeController is 'function'

  nameEndsWithController: ->
    ...